src/PhpUnit/UriInterfaceTestsTrait.php

```diff
@@ -0,0 +1,290 @@
+<?php
+
+namespace Maks3w\Psr7Assertions\PhpUnit;
+
+use PHPUnit_Framework_Assert as Assert;
+use Psr\Http\Message\UriInterface;
+
+/**
+ * Provide PHPUnit test methods for Psr\Http\Message\UriInterface constraints.
+ */
+trait UriInterfaceTestsTrait
+{
+    /**
+     * @return Psr\Http\Message\UriInterface
+     */
+    abstract public function createUriObject($url);
```

Comment:
My first intention is to test a default object is PSR7 compliant inmediatelly after object instantiation.

For this reason this abstract method is named "create**Default**Uri" in the rest of traits

Please refactor the name and remove the argument